labs/sawtooth-healthcare.md

```diff
@@ -0,0 +1,30 @@
+# Lab Name
+Sawtooth Healthcare.
+
+# Short Description
+
+Sawtooth Healthcare is a blockchain application built on Hyperledger Sawtooth, allowing clinics to keep history regarding patients health and claims on the blockchain.
+
+# Scope of Lab
+
+The scope of the lab is to develop a proof of concept (POC) for a blockchain-based solution that covers most common use case in healthcare. 
+
+Also, this repo can be used for practical needs with Sawtooth blockchain since it covers/will cover most of components required to build dApp.
+
+In addition I plan to add some load tests and provide comparative matrix and summary report on how various aspects/tuning may vary on performance/throughput.
+
+# Initial Committers
+
+For now I'm the only person. Fill free to join and expand the community.
+
+https://github.com/AlexZhovnuvaty Alexander Zhovnuvaty <a.zhovnuvaty@gmail.com>
+
+# Sponsor
+
+Richard Bloch <richardbloch@businesslearninginc.com>
```

Comment:
Add `, chair of Healthcare SIG` after email address.

Reply:
Updated in recent commit